Spotify-Swift/SpotifyArtistsViewer/SASearchViewController.swift

```diff
@@ -48,9 +48,9 @@ class SASearchViewController: UITableViewController, UITextFieldDelegate {
     }
     
     override internal func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCellWithIdentifier("cell", forIndexPath: indexPath) as cellType
```

Comment:
Must've snuck in when I kept a copy of the Swift project in this branch. I'll take it out back...